tests/unittest_inference.py

```diff
@@ -6628,38 +6628,60 @@ def test_inference_of_items_on_module_dict() -> None:
     builder.file_build(str(DATA_DIR / "module_dict_items_call" / "test.py"), "models")
 
 
-class ImportedModuleTests(resources.AstroidCacheSetupMixin):
```

Comment:
Just a note, you don't need to remove test classes. Even with `pytest`, they can be helpful to group test cases. I believe the only important things to remember are the classname must begin with `Test...` and you can use `@staticmethod` if you don't need `self`.